bool moreThanOne
bool biggerOnTop
bool oneLess
